model-optimizer/mo/graph/port.py

```diff
@@ -278,7 +278,7 @@ def get_destinations(self):
             consumer_ports.append(node.in_port(d['in'], control_flow=self.control_flow))
         return consumer_ports
 
-    def get_tensor_names(self, port_renumber: bool = False):
+    def get_tensor_names(self, port_renumber: bool = False, first_only: bool = False):
```

Comment:
I would recommend to change the semantic of this function to return a list of node names instead of a concatenated string with node names. And the function which uses these tensor names should concatenate them if necessary.